src/openrct2/windows/TopToolbar.cpp

```diff
@@ -1978,7 +2001,9 @@ static void top_toolbar_tool_update_land(sint16 x, sint16 y){
         return;
     }
 
-    screen_get_map_xy(x, y, &mapTile.x, &mapTile.y, nullptr);
+    screen_get_map_xy_side(x, y, &mapTile.x, &mapTile.y, &side); //reuse old variables, we do not need dummies here
+
+    //screen_get_map_xy(x, y, &mapTile.x, &mapTile.y, nullptr);
```

Comment:
Delete code, do not comment it out. We have source control to get old code back.